### ISSUE TITLE
Eloquent with GlobalScope have Exception

### ISSUE THREAD

**Issue by dryyun** [damiantw/laravel-scout-mysql-driver#10]
I have a model ``Post`` use ``GlobalScope`` 
```
    protected static function boot()
    {
        parent::boot();

       static::addGlobalScope('status', function(Builder $builder) {
            $builder->where('status', '=', 1);
        });

    }
```
And use ``laravel-scout-mysql-driver`` for search  q=``dfds``

It occur  QueryException  
``select count(*) as aggregate from `posts` where (`content` LIKE :_search0) and `posts`.`deleted_at` is null and `status` = %dfds%)``

**Comment by WellsMediaGroup** [comment]
I believe the issue is the mixing of named and positional parameters.  Eloquent uses positional and this lib uses named.

If you are using the Like comparison, change

```$queryString .= "`$field` LIKE :_search$itr OR ";```
to
```$queryString .= "`$field` LIKE ? OR ";```

**Comment by ChrisThompsonTLDR** [comment]
I believe the issue is the mixing of named and positional parameters.  Eloquent uses positional and this lib uses named.

If you are using the Like comparison, change

```$queryString .= "`$field` LIKE :_search$itr OR ";```
to
```$queryString .= "`$field` LIKE ? OR ";```

**Comment by c4l3b** [comment]
Fixed in #31
